### PR TITLE
DDF-3706 Fix list render bug

### PR DESCRIPTION
**Pull request by Bdthomson**
#### What does this PR do?
Fixes a bug introduced by #3112 where lists with elements in them do not render.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)

@andrewkfiedler 
@mojogitoverhere 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@bdeining

#### What are the relevant tickets?
[DDF-3706](https://codice.atlassian.net/browse/DDF-3706)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/workspace-lists/workspace-lists.view.js [hunk 1]
```diff
@@ -123,7 +123,8 @@ module.exports = Marionette.LayoutView.extend({
             selectedListId = listId;
             if (!this.model.get(selectedListId).isEmpty()) {
                 this.listResults.show(new ResultSelectorView({
-                    model: this.model.get(selectedListId).get('query')
+                    model: this.model.get(selectedListId).get('query'),
+                    selectionInterface: store.get('content')
```
Comment (mackncheesiest) [inline ui/packages/catalog-ui-search/src/main/webapp/component/workspace-lists/workspace-lists.view.js]: ❓ I think you just want to pass `store` in here

Comment (andrewkfiedler) [reply ui/packages/catalog-ui-search/src/main/webapp/component/workspace-lists/workspace-lists.view.js]: ✏️ ☝️

**Comment by bdeining** [comment]
build now

**Pull request merged**